funcx_endpoint/funcx_endpoint/endpoint/results_ack.py

```diff
@@ -70,5 +75,12 @@ def get_unacked_results_list(self):
     def persist(self):
         """ Save unacked results to disk
         """
-        # TODO: pickle dump unacked_results
-        return
+        with open(self.data_path, 'wb') as fp:
```

Comment:
Are we relying on empty file to test whether persist worked? If not we should dump only if `unacked_results` is non-empty.

Reply:
Hmm, I'm not sure I agree because if the data file `unacked_results.p` currently has old unacked results sitting in it, but those results were since acked when `persist` is called, then the file should either be overwritten as empty or it should be deleted. Simply overwriting with empty is the easiest approach

Reply:
Okay, that makes sense.